docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,169 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
+## How to use predefined entering animation?
+
+### 1. Import chosen animation
+```js
+    import { Animation } from 'react-native-reanimated';
+```
+### 2. Choose Animated Component which entering you want to animate
+```js
+    <AnimatedComponent entering={Animation} >
+```
+### 3. Customize the animation
+    If you don't know what modifiers the animation provides then find your animation down below.
+```js
+    <AnimatedComponent entering={Animation.duration(3000).otherModifier()} >
+```
+### 4. Make sure that your animated component is under an AnimatedLayout. If it's not then add AnimatedLayout somewhere above the component.
+```js
+    <AnimatedLayout> // +
+        <View> sth </View>
+        <View> 
+            <AnimatedComponent entering={Animation}>
+        </View>
+    </AnimatedLayout> // +
+```
+
+## Predefined Animations 
+If you cannot find an animation that suits you then please create your custom animation. If you think that the animation should be here, please open an issue or create a pull request. 
```

Comment:
This paragraph is titled "predefined animations" but actually talks about "custom animations"

Reply:
Also "please create your custom animation" sounds very weird. I'd write "you can create a custom one"